redirects.json

```diff
@@ -0,0 +1,47 @@
+[
```

Comment:
my only thoughts here are that maybe we should add an exhaustive list if this works smoothly. however, i can't see them 😅 
<img width="1446" height="1773" alt="Screenshot 2025-12-09 at 11 46 36" src="https://github.com/user-attachments/assets/d60fd634-0eb6-4a2b-8a37-2008741faa16" />